client/app/account/user/security/user-security.html

```diff
@@ -81,14 +81,14 @@
                                 data-ng-if="!has2faEnabled()">
                             <p class="text-warning"
                                 data-translate="user_account_security_double_auth_info"></p>
-                            <button type="button"
-                                    class="oui-button oui-button_primary float-right"
-                                    data-ng-click="setAction('security/2fa/user-security-2fa')"
-                                    data-track-on="click"
-                                    data-track-name="account::security::authentication::double_authentication"
-                                    data-track-type="action">
+
+                            <a data-ui-sref="app.account.user.security.mfa"
```

Comment:
Shouldn't be `app.account.user.security.2fa`? (Refer to my previous comment).

Reply:
It should be mfa everywhere. It's multiple factor authentication, there's no restriction to only two factors.